crates/bevy_gltf/Cargo.toml

```diff
@@ -11,6 +11,7 @@ keywords = ["bevy"]
 [features]
 dds = []
 pbr_transmission_textures = []
+pbr_multi_layer_material_textures = []
```

Comment:
I feel like we should split StandardMaterial into 2 / features / something. A basic one with the typical PBR stuff for lower end platforms, and one with all features included for newer platforms. Doesn't have to be this PR.

Reply:
Yeah, I considered doing that here, but decided against it because of the precedent set by transmission, which is also a more advanced feature. My personal feeling is that maybe `StandardMaterial` should be the glTF standard with no extensions, and then we can have `GltfExtensionMaterial`, `OpenPbrMaterial`, `MaterialXMaterial` that build on top of it. But I imagine there will be a lot of discussion around this topic, so that's for a followup :)